File: CS_UBB_BACHELORS/POO/POO Laboratory 11-12/Lab11-12/Lab11-12/AppStart.cpp
#include"domain.h"
#include<stdio.h>
#include "repo.h"
#include "controller.h"
#include "UI.h"
#include<windows.h>
#include<assert.h>
#include"tests.h"


int main() {
	//Title Genre Trailer Year Likes
	UI ui;
	tests testing;
	ui.mainLoop();
	return 0;
}

File: CS_UBB_BACHELORS/ag/Lab 5/AppStart.py
from domain.UndirectedGraph import Graph
from Repository.Repository import Repository
from controller.Controller import Controller
from ui.UI import UI
graph = Graph()
repo = Repository(graph)
repo.loadFromFile()
controller = Controller(repo)
ui = UI(controller)
#print(ui._controller.getAllVertices())
ui.mainLoop()
#print(ui._controller.biconnectedGraph())
